javascript/javascript1/week1/homework.js

```diff
@@ -0,0 +1,86 @@
+// Age-ify (A future age calculator)
+const yearOfBirth = 1999;
+const yearFuture = 2050;
+const age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture);
+
+// Goodboy-Oldboy (A dog age calculator)
+const dogYearOfBirth = 2020;
+const dogYearFuture = 2027;
+const dogYear = dogYearFuture - dogYearOfBirth;
+const shouldShowResultInDogYears = true;
+if (shouldShowResultInDogYears) {
+  console.log(
+    "Your dog will be " + dogYear * 7 + " dog years old in " + dogYearFuture
+  );
+} else if (!shouldShowResultInDogYears) {
```

Comment:
```suggestion
} else {
```
If the code in the `if` part is not true, then this part will be executed. No need to be explicit with `else if (!shouldShowResultInDogYears)`